src/utils/common.js

```diff
@@ -1,10 +1,13 @@
 import * as R from "ramda";
 
 export const getAnchorPart = (anchor, index) => {
-  return R.compose(
-    R.view(R.lensIndex(index)),
-    R.split(".")
-  )(anchor);
+  if (anchor) {
+    return R.compose(
+      R.view(R.lensIndex(index)),
+      R.split(".")
+    )(anchor);
+  }
+  return {};
```

Comment:
Type of the return value is string if the anchor is defined otherwise an object. I would prefer to fix it like this:

```
const getAnchorPart = (anchor, index) => {
  return R.compose(
    R.view(R.lensIndex(index)),
    R.split(".")
  )(anchor || "");
};
```
It would return undefined if a value of the anchor is falsy.

Reply:
Why not.